src/Microsoft.TemplateEngine.Core/Conditional.cs

```diff
@@ -308,6 +472,25 @@ public bool BranchTaken
                     get { return _branchTaken; }
                     set { _branchTaken |= value; }
                 }
+
+                public bool ActionableOperationsEnabled { get; private set; }
+
+                public void ToggleActionableOperations(bool enabled, IProcessorState processor)
+                {
+                    ActionableOperationsEnabled = enabled;
+
+                    foreach (string otherOptionDisableFlag in _impl._definition._actionableOperations)
```

Comment:
processor.Config.Flags[otherOptionDisableFlag] = enabled;

instead of contains/add vs set